vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -83,13 +83,18 @@ echo "Granting access to $disk_resource_id for 1 hour"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
 
-# TBD: Need to investigate why `azcopy-preview login --login-type=MSI` does not work
-echo "Setting azcopy environment variables with pool identity: $AZURE_MSI_RESOURCE_STRING"
-export AZCOPY_AUTO_LOGIN_TYPE="MSI"
-export AZCOPY_MSI_RESOURCE_STRING="$AZURE_MSI_RESOURCE_STRING"
-
 echo "Uploading $disk_resource_id to ${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd"
-azcopy-preview copy "${sas}" "${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd" --recursive=true || exit $?
+
+if [[ "${OS_TYPE}" == "Linux" ]]; then
+  azcopy login --login-type=MSI
```

Comment:
so this isn't working in the windows case?

Reply:
I don't believe the windows AzCopy is a current enough version. I am running some pipelines right now to double check and see a couple different methods. Back when I was working on the shallow replication PR, this was the only way I could get things to work without the container. For some reason, I can not seem to get auto-login to work with just the agent even though the resource string is still passed in.

Reply:
So seems that I need to reach out to the windows team to check on their pipeline, I have to set SIG_FOR_PRODUCTION to true in order for the convert script to run. Would like to do speak with them before doing so to make sure I don't inadvertently cause issues @cameronmeissner

Reply:
I did not leave the TODO comment by the way regarding --login-type=MSI, that was Chou Hu